libs/coin-modules/coin-ton/src/utils.ts

```diff
@@ -212,7 +214,9 @@ export const getTonEstimatedFees = async (
     initCode,
     initData,
   );
-  return BigNumber(fee.fwd_fee + fee.gas_fee + fee.in_fwd_fee + fee.storage_fee);
+  return isJetton
```

Comment:
```suggestion
  return tx.payload?.type === "jetton-transfer"
```
To avoid declaring another variable.